client/app/bundles/HelloWorld/startup/registration_BACKUP_10665.jsx

```diff
@@ -0,0 +1,16 @@
+import ReactOnRails from 'react-on-rails';
```

Comment:
This file, as well as some other artifacts of merge conflicts (and the `swp` files), shouldn't have been added to the repo.